module/behaviour/planning/KickPlanner/src/KickPlanner.cpp

```diff
@@ -50,12 +52,18 @@ namespace behaviour {
         using message::behaviour::KickPlan;
         using KickType = message::behaviour::KickPlan::KickType;
         using message::behaviour::WantsToKick;
+        using message::input::GameState;
         using message::input::Sensors;
         using message::localisation::Ball;
         using message::localisation::Field;
+        using Phase         = message::input::GameState::Data::Phase;
+        using PenaltyReason = message::input::GameState::Data::PenaltyReason;
```

Comment:
wouldn't `using message::input::GameState::Data::Phase` have the same effect as the explicit assignment of the alias? 
same for the line after
```suggestion
        using message::input::GameState::Data::Phase;
        using message::input::GameState::Data::PenaltyReason;
```